src/main/java/org/folio/inventory/storage/external/repository/RequestRepository.java

```diff
@@ -0,0 +1,54 @@
+package org.folio.inventory.storage.external.repository;
+
+import static java.util.Arrays.asList;
+import static org.folio.inventory.domain.view.request.RequestStatus.OPEN_AWAITING_DELIVERY;
+import static org.folio.inventory.domain.view.request.RequestStatus.OPEN_AWAITING_PICKUP;
+import static org.folio.inventory.domain.view.request.RequestStatus.OPEN_IN_TRANSIT;
+import static org.folio.inventory.storage.external.CqlQuery.exactMatch;
+import static org.folio.inventory.storage.external.CqlQuery.exactMatchAny;
+import static org.folio.inventory.storage.external.Limit.one;
+import static org.folio.inventory.storage.external.Offset.noOffset;
+import static org.folio.inventory.support.JsonArrayHelper.toList;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+
+import org.folio.inventory.domain.view.request.Request;
+import org.folio.inventory.storage.external.Clients;
+import org.folio.inventory.storage.external.CollectionResourceRepository;
+import org.folio.inventory.storage.external.CqlQuery;
+
+public class RequestRepository {
+  private static final List<String> REQUEST_IN_FULFILLMENT_STATUSES =
+    getRequestInFulfillmentStatuses();
```

Comment:
Why is this needed as a static constant, rather than being determined when it is needed?

Reply:
I've removed the constant.